src/main/java/com/conveyal/gtfs/GTFS.java

```diff
@@ -159,12 +202,17 @@ public static void main (String[] args) {
             return;
         }
 
-        if (!(cmd.hasOption("export") || cmd.hasOption("snapshot") || cmd.hasOption("load") || cmd.hasOption("validate") || cmd.hasOption("graphql"))) {
-            LOG.error("Must specify one of 'snapshot', 'load', 'validate', 'export', or 'graphql'.");
+        if (!(cmd.hasOption("export") || cmd.hasOption("snapshot") || cmd.hasOption("load") || cmd.hasOption("validate") || cmd.hasOption("delete"))) {
+            LOG.error("Must specify one of 'snapshot', 'load', 'validate', 'export', or 'delete'.");
             printHelp(options);
             return;
         }
-
+        boolean storeResults = cmd.hasOption("json");
+        File directory = null;
```

Comment:
I would move the creation of the objectMapper down here, conditional upon storeResults=true.